src/lib/utils/thorchain/lp.ts

```diff
@@ -201,7 +181,7 @@ export const estimateRemoveThorchainLiquidityPosition = async ({
     runeAmountCryptoThorPrecision,
   })
 
-  const poolShare = getPoolShare(liquidityUnitsCryptoThorPrecision, pool)
+  const poolShare = getPoolShare(pool, liquidityUnitsCryptoThorPrecision)
```

Comment:
Can we grep all those `thorPrecision` and make them `thorBaseUnit`? 🙏🏽 precision amounts refer to full-precision, not base units, so this vernacular really doesn't make sense

Reply:
yessir, changes coming in next PR!